.github/workflows/artillery-tests.yaml

```diff
@@ -1,24 +1,33 @@
-name: Artillery Users Test
-
-on:
-  workflow_run:
-    workflows: ["Deploy Function App to Azure"]
-    branches: 
-      - development
-      - derek-task1-sprint5
-    types:
-      - completed
+name: Artillery Tests
 
 #on:
-#  push:
-#    branches:
+#  workflow_run:
+#    workflows: ["Deploy Function App to Azure"]
+#    branches: 
 #      - development
 #      - derek-task1-sprint5
```

Comment:
We probably do not need to keep - derek-task1-sprint5 you used for testing in to development